pom.xml

```diff
@@ -24,46 +30,14 @@
     </license>
   </licenses>
 
-  <distributionManagement>
-    <repository>
-      <id>maven.jenkins-ci.org</id>
-      <url>http://maven.jenkins-ci.org:8081/content/repositories/releases/</url>
-    </repository>
-  </distributionManagement>
-
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <java.level>6</java.level>
```

Comment:
Do we really anyway want to care about this, let's bump to Java 8? Though I understand this might make the change not scoped enough.

Reply:
I will leave it to somebody who touches this repo next time